### ISSUE TITLE
state of debugger

### ISSUE THREAD

**Issue by BlueHotDog** [elixir-lsp/elixir-ls#243]
### Environment

* Elixir & Erlang versions (elixir --version): Elixir 1.10.3 (compiled with Erlang/OTP 22)
* Operating system: OSX
* Editor or IDE name (e.g. Emacs/VSCode): VScode
* LSP Client name: 
  * If using VSCode, are you using "ElixirLS: Elixir support and debugger"? yes

### Troubleshooting

- [x] Restart your editor (which will restart ElixirLS) sometimes fixes issues
- [ ] Stop your editor, remove the entire `.elixir_ls` directory, then restart your editor
  * NOTE: This will cause you to have to re-run the dialyzer build for your project

### Logs

```
11:42:59.570 [error] Process #PID<0.116.0> raised an exception
** (MatchError) no match of right hand side value: {:error, :on_load_failure}
    int.erl:531: anonymous fn/3 in :int.load/2
    int.erl:527: :int.load/2
    (elixir 1.10.3) lib/enum.ex:1396: Enum."-map/2-lists^map/1-0-"/2
    (elixir 1.10.3) lib/enum.ex:1396: Enum."-map/2-lists^map/1-0-"/2
    (debugger 0.3.3) lib/debugger/server.ex:477: ElixirLS.Debugger.Server.initialize/1
```

When using the following vscode task
```
{
        "type": "mix_task",
        "request": "launch",
        "name": "mix phx.server",
        "task": "phx.server",
        "taskArgs": [
            "--trace"
        ],
        "projectDir": "${workspaceRoot}",
        "excludeModules": [":bcrypt_nif"]
    }
```

**Comment by lukaszsamson** [comment]
It looks like one of your NIF fails to load. Is it compiled for the correct version of OTP (`mix deps.clean` it to be sure).
Anyway, there are similar reports https://github.com/KronicDeth/intellij-elixir/issues/681

**Comment by lukaszsamson** [comment]
I did some testing and the debugger seems to work when I add all NIFs to `excludeModules`. @axelson I think we should add an error message showing which modules caused the failure.

**Comment by axelson** [comment]
Yeah that makes sense :+1:

**Comment by axelson** [comment]
Should be fixed in #283
